test.js

```diff
@@ -1,19 +1,20 @@
-'use strict';
-var path = require('path');
-var test = require('ava');
-var fn = require('./');
-var cwd = path.join(__dirname, 'fixture');
-var pkgPath = path.join(__dirname, 'package.json');
+import test from 'ava';
+import path from 'path';
+import fn from './';
 
-test('async', function (t) {
-	return fn({cwd: cwd}).then(function (x) {
-		t.is(x.pkg.name, 'read-pkg-up');
-		t.is(x.path, pkgPath);
-	});
+const cwd = path.join(__dirname, 'fixture');
+const pkgPath = path.join(__dirname, 'package.json');
+
+test('async', async t => {
+	const x = await fn({cwd: cwd});
```

Comment:
We can use [destructuring assignment](https://github.com/lukehoban/es6features#destructuring). 

```js
const x = await fn({cwd});
```